### PR TITLE
Fix: Fix bugs for SVG Editor

### PR DESCRIPTION
**Pull request by meiliang1123**
- Set new text at center of canvas
- Update source of rectangle and ellipse when resizing
- Remove code of moving configuration to series folder

### CODE REVIEW

File: src/app/flux/models/SvgModel.js [hunk 1]
```diff
@@ -391,7 +391,7 @@ class SvgModel {
         };
         // reserve scale cause stroke-width scaled
         if (this.type === 'rect' || this.type === 'ellipse') {
-            attrs.config['stroke-width'] = 0.25 / Math.min(scaleX, scaleY);
+            // attrs.config['stroke-width'] = 0.25 / Math.min(scaleX, scaleY);
```
Comment (zjiefee) [inline src/app/flux/models/SvgModel.js:394]: 这个是什么 如果要注释 不把 if 也注释了吗

File: src/app/widgets/CncLaserShared/Transformation.jsx [hunk 2]
```diff
@@ -270,13 +270,18 @@ const mapStateToProps = (state, props) => {
     const machine = state.machine;
     const { modelGroup } = state[headType];
     const selectedModel = modelGroup.getSelectedModel();
-    const { modelID, sourceType, visible, transformation } = selectedModel;
+    const { modelID, sourceType } = selectedModel;
+    const transformation = { ...selectedModel.transformation };
+    if (sourceType === 'raster') {
+        transformation.width *= selectedModel.transformation.scaleX;
+        transformation.height *= selectedModel.transformation.scaleY;
```
Comment (zjiefee) [inline src/app/widgets/CncLaserShared/Transformation.jsx:277]: width height 的含义变了吗

File: src/server/DataStorage.js [hunk 3]
```diff
@@ -178,52 +178,52 @@ class DataStorage {
...
-                     } finally {
-                         // Regardless copy success or not, remove the configuration file
-                         fs.unlinkSync(src);
-                     }
-                 }
-             }
-         }
-         log.info('Migration done.');
-     }
+    //  async versionAdaptation() {
```
Comment (parachvte) [inline src/server/DataStorage.js:181]: We can remove the method completely?

**Comment by jane-rose** [comment]
#402

**Pull request merged**